### PR TITLE
Adjust needle matching strategy for xen host in reboot_and_wait_up

### PR DESCRIPTION
**Pull request by waynechen55**
* **SOL** session output for xen host may produce additional messages which may disrupt what is displayed on sol console and then the needle matching, for example, [this may happen on xen host](https://openqa.suse.de/tests/7619536#step/reboot_and_wait_up_upgrade/25).

* **So** xen host needs special treatment to get the login prompt back on screen and then match 'linux-login' needle if 'text-login' can not be done successfully.

* **Remove** duplicate "use ipmi_backend_utils;"

* **Verification runs:**
  * [sles12sp5 to sles15sp4 xen](https://openqa.suse.de/tests/7609439)
  * [sles15sp3 to sles15sp4 xen](https://openqa.suse.de/tests/7609440)

### CODE REVIEW

**Comment by waynechen55** [comment]
@alice-suse @Julie-CAO @guoxuguang @nanzhg Please have a look.

**Comment by guoxuguang** [comment]
LGTM

**Comment by nanzhg** [comment]
LGTM

File: tests/virt_autotest/reboot_and_wait_up.pm [hunk 1]
```diff
@@ -59,7 +58,16 @@ sub reboot_and_wait_up {
             #login is required when sol console is used for the first time
             unless (check_screen('text-logged-in-root')) {
                 #The timeout can't be too small since autoyast installation
-                assert_screen "text-login", 600;
+                #Xen console may output additional messages about vm on sol whose output is disrupted.
+                #So in order to get login prompt back on screen, 'ret' key should be fired up. But the
+                #os name banner might be available anymore, only 'linux-login' needle can be matched.
```
Comment (Julie-CAO) [inline tests/virt_autotest/reboot_and_wait_up.pm:63]: missing 'not' after 'might' in the comment?

**Comment by Julie-CAO** [comment]
Sorry for late review. I verified in xen tests for opensuse TW, everything looks fine. http://10.67.129.54/tests/34

File: tests/virt_autotest/reboot_and_wait_up.pm (hunk 1 above)
Comment (waynechen55) [reply tests/virt_autotest/reboot_and_wait_up.pm]: Updated.

**Comment by alice-suse** [comment]
Seems all are good with it. Let it go.

**Pull request merged**